laika/raw_gnss.py

```diff
@@ -200,7 +202,7 @@ def read_raw_qcom(report):
       #print("  %.5f %3d %10.2f %7.2f %7.2f %.2f %d" % (recv_time.tow, i.svId,
       #  observables['C1C'], observables_std['C1C'],
       #  observables_std['D1C'], observables['S1C'], i.latency), i.observationState, i.measurementStatus.fineOrCoarseVelocity)
-      glonass_freq = (i.glonassFrequencyIndex - 7) if report.source == 1 else np.nan
+      glonass_freq = (i.glonassFrequencyIndex - 7) if report.source == 1 else None
```

Comment:
Changed glonass_freq to be Optional[int]. However not sure if this change will break anything in private repo?

Reply:
Discussed with Willem. Created an issue for it to do it another time since a change in the private repo is need too: https://github.com/commaai/laika/issues/74